.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
Please don't modify this file

Reply:
I have not edited this file. Nor opened it, to be honest.